### PR TITLE
feat(codecov): walk back 20 commits instead of 10

### PR DESCRIPTION
**Pull request by armenzg**
There's a chance that some customers may benefit if they commit a lot in a day and may have had few hours of CI outage or bad config.

### CODE REVIEW

**Comment by armenzg** [comment]
I would have to adjust the widgets.

**Comment by armenzg** [comment]
I will wait for #46257 to merge first and analyzed before merging this as I don't want to pollute the data.

**Review by snigdhas — APPROVED** [review]

**Pull request closed without merging**
